.circleci/config.yml

```diff
@@ -29,3 +32,33 @@ jobs:
           key: dependency-cache-{{ checksum "package.json" }}
           paths:
             - node_modules
+
+  # TODO: Don't recompile everything. Use workspace persistance.
+  deploy:
+    <<: *defaults
+    description: "Publish package to npmjs.org"
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run: yarn install
+      - run: yarn build
+      - run:
+          name: Authenticate with NPM
+          command: 'echo "//registry.npmjs.org/:_authToken=$NPM_PUBLISH_TOKEN" > ~/lightning-language-server/.npmrc'
+      - run:
+          name: Publish to NPM
+          command: yarn publish-lsp
```

Comment:
I think we need to commit changes after this right because the versions will get bumped during this process

Reply:
Lerna does it automatically.